svir/test/integration/test_drive_oq_engine.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+# /***************************************************************************
+# Irmt
+#                                 A QGIS plugin
+# OpenQuake Integrated Risk Modelling Toolkit
+#                              -------------------
+#        begin                : 2014-10-24
+#        copyright            : (C) 2014-2017 by GEM Foundation
+#        email                : devops@openquake.org
+# ***************************************************************************/
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake.  If not, see <http://www.gnu.org/licenses/>.
+
+# import qgis libs so that we set the correct sip api version
+import os
+import unittest
+import time
+import tempfile
+import zipfile
+import json
+import numpy
+
+from svir.third_party.requests import Session
+from svir.utilities.utils import listdir_fullpath
+
+
+class DriveOqEngineTestCase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.session = Session()
+        # FIXME: it might be passed as argument when running integration tests
+        cls.demos_dir = os.path.join(
+            os.path.expanduser('~'), 'projects', 'oq-engine', 'demos')
```

Comment:
I would pass it as an argument and `oq-engine/demos` (relative to cwd) as default